src/extension/intellisenseHelper.ts

```diff
@@ -182,7 +182,7 @@ export class IntellisenseHelper {
             let setEnvironmentVariableCommand: string = "";
             if (os.type() === "Darwin") {
                 setEnvironmentVariableCommand = "launchctl setenv VSCODE_TSJS 1";
-            } else if (os.type() === "Windows") {
+            } else if (os.type() === "Windows_NT") {
                 setEnvironmentVariableCommand = "setx VSCODE_TSJS 1";
             }
```

Comment:
seems like we have this problem throughout this class (not sure about others) - should all be consolidated in the platform abstraction and cleaned up to support all three platforms